apps/condo/jest.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
             testMatch: [
                 `${__dirname}/schema/**/*.test.js`,
                 `${__dirname}/domains/**/schema/*.test.js`,
+                `${__dirname}/domains/**/tasks/*.test.js`,
```

Comment:
It's better not to add a new path here. Usualy you need to move your tests inside [BankSyncTask.test.js](https://github.com/open-condo-software/condo/pull/3044/files#diff-64a39bd8e684e8d512835b78516654600c77d1965c75aabfeb2366a1e1a4d5c4)